packages/nodes-base/nodes/CustomerIo/GenericFunctions.ts

```diff
@@ -0,0 +1,48 @@
+import {
+	IExecuteFunctions,
+	IHookFunctions,
+	ILoadOptionsFunctions,
+} from 'n8n-core';
+
+import { OptionsWithUri } from 'request';
+import { IDataObject } from 'n8n-workflow';
+
+export async function apiRequest(this: IHookFunctions | IExecuteFunctions | ILoadOptionsFunctions, method: string, endpoint: string, body: object, query?: IDataObject): Promise<any> { // tslint:disable-line:no-any
```

Comment:
to go w codebase conventions, name this `customerIoApiRequest`

Reply:
ya, I'll make the changes